src/Security/Security.php

```diff
@@ -764,14 +1035,30 @@ public function getCredentials($strategy, $kuid, array $options = [])
      * @param $strategy
      * @param $kuid
      * @param array $options
+     *
      * @return mixed
+     *
+     * @throws InvalidArgumentException
      */
     public function getCredentialsById($strategy, $kuid, array $options = [])
```

Comment:
The user id provided to the `getCredentialsById` is NOT the kuid, but the strategy's specific user id (for instance, the "username" for the `local` strategy)

See: https://docs.kuzzle.io/api-documentation/controller-security/get-credentials-by-id